### ISSUE TITLE
Getting Malware warning on Ventura M1 while installing

### ISSUE THREAD

**Issue by khurshid-alam** [kotleni/FinderPlus-macOS#2]
Getting this warning . 

<img width="341" alt="CleanShot 2024-07-05 at 17 08 24" src="https://github.com/kotleni/FinderPlus-macOS/assets/1333217/b0b15b43-742f-416f-85cd-03c498cad070">

`xattr -c /Applications/FinderPlus.app` didn't work. Cert issue ?

**Comment by kotleni** [comment]
@khurshid-alam Yeah, it's a certificate issue. Just open app by RIGHT MOUSE BUTTON -> Open.

**Comment by khurshid-alam** [comment]
@kotleni That didn't work for me. I managed to disable the malware warning window by using,

`codesign --force --deep --sign - /Applications/FinderPlus.app`

But now nothing shows up in Extension menu in the preference. Does it work in apple silicon (Ventura) ? Can you confirm ?

**Comment by kotleni** [comment]
> @kotleni That didn't work for me. I managed to disable the malware warning window by using,
> 
> `codesign --force --deep --sign - /Applications/FinderPlus.app`
> 
> But now nothing shows up in Extension menu in the preference. Does it work in apple silicon (Ventura) ? Can you confirm ?

Idk, you can try rebuild it by youself from XCode and run it without codesign.
I have worked it on my Macbook Air M1
